lib/streamlit/elements/widgets/audio_input.py

```diff
@@ -91,20 +91,84 @@ def audio_input(
         key: Key | None = None,
-        help: str | None = None,
         on_change: WidgetCallback | None = None,
         args: WidgetArgs | None = None,
         kwargs: WidgetKwargs | None = None,
         *,  # keyword-only arguments:
         disabled: bool = False,
         label_visibility: LabelVisibility = "visible",
     ) -> UploadedFile | None:
-        """TODO[kajarenc]ADD DOCSTRINGS HERE"""
+        r"""Display a widget that returns audio recording from the user's microphone.
+
+        Parameters
+        ----------
+        label : str
+            A short label explaining to the user what this widget is used for.
+            The label can optionally contain GitHub-flavored Markdown of the
+            following types: Bold, Italics, Strikethroughs, Inline Code, and
+            Links.
+
+            Unsupported Markdown elements are unwrapped so only their children
+            (text contents) render. Display unsupported elements as literal
+            characters by backslash-escaping them. E.g.,
+            ``"1\. Not an ordered list"``.
+
+            See the ``body`` parameter of |st.markdown|_ for additional,
+            supported Markdown directives.
+
+            For accessibility reasons, you should never set an empty label (label="")
+            but hide it with label_visibility if needed. In the future, we may disallow
+            empty labels by raising an exception.
+
+            .. |st.markdown| replace:: ``st.markdown``
+            .. _st.markdown: https://docs.streamlit.io/develop/api-reference/text/st.markdown
+
+        key : str or int
+            An optional string or integer to use as the unique key for the widget.
+            If this is omitted, a key will be generated for the widget
+            based on its content. Multiple widgets of the same type may
+            not share the same key.
+
+        on_change : callable
+            An optional callback invoked when this camera_input's value
+            changes.
+
+        args : tuple
+            An optional tuple of args to pass to the callback.
+
+        kwargs : dict
+            An optional dict of kwargs to pass to the callback.
+
+        disabled : bool
+            An optional boolean, which disables the camera input if set to
+            True. Default is False.
+        label_visibility : "visible", "hidden", or "collapsed"
+            The visibility of the label. If "hidden", the label doesn't show but there
+            is still empty space for it above the widget (equivalent to label="").
+            If "collapsed", both the label and the space are removed. Default is
+            "visible".
+
+        Returns
+        -------
+        None or UploadedFile
+            The UploadedFile class is a subclass of BytesIO, and therefore
+            it is "file-like". This means you can pass them anywhere where
+            a file is expected.
+
+        Examples
+        --------
+        >>> import streamlit as st
```

Comment:
out of curiosity, do we run these as doctests?

Reply:
@sfc-gh-nbellante I think we don't actually run them; the only reason for this is to have it as a code block in our documentation, @sfc-gh-dmatthews probably knows better.

Reply:
👌

Reply:
Correct. Examples in docstrings are used for docs. I typically follow up on engineering work to modify or add examples, deploy them to Community Cloud, and insert the live embedded app to render in docs.